src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseFileServicePermissionTests.cs

```diff
@@ -132,9 +167,15 @@ await PolicyCheckBuilder<SecurityPolicies>()
                     userService =>
                     {
                         var service = SetupReleaseFileService(userService: userService.Object);
-                        return service.UploadAncillary(releaseId: _release.Id,
-                            formFile: new Mock<IFormFile>().Object,
-                            name: "");
+                        return service.UploadAncillary(
+                            releaseId: _release.Id,
+                            upload: new ReleaseAncillaryFileUploadViewModel
+                            {
+                                File = Mock.Of<IFormFile>(),
```

Comment:
Didn't know of this short-hand before.  Worth mentioning in #development channel